packages/api-docs-builder/utils/generateApiTranslation.ts

```diff
@@ -0,0 +1,52 @@
+import { mkdirSync } from 'fs';
+import path from 'path';
+import kebabCase from 'lodash/kebabCase';
+import { writePrettifiedFile } from '../buildApiUtils';
+
+interface MinimalReactAPI {
+  name: string;
+  apiDocsTranslationFolder?: string;
+  translations: object;
+}
+
+export default function generateApiTranslations<ReactApi extends MinimalReactAPI>(
+  outputDirectory: string,
+  reactApi: ReactApi,
+  languages: string[],
+) {
+  const hookName = reactApi.name;
```

Comment:
```suggestion
  const definitionName = reactApi.name;
```

It can be both hook and component.